app/Http/Controllers/commentController.php

```diff
@@ -2,23 +2,49 @@
 
 namespace App\Http\Controllers;
 
-use App\Services\TaskService;
+use App\Services\TaskCommentService;
 use Illuminate\Http\Request;
+use Illuminate\Support\Carbon;
 
-class commentController extends Controller
+class CommentController extends Controller
 {
+    protected $taskCommentService;
 
-    public function __construct(){}
+    public function __construct(TaskCommentService $taskCommentService)
+    {
+        $this->taskCommentService = $taskCommentService;
+    }
 
+    public function index()
+    {
+    }
 
+    public function store(Request $request)
+    {
 
-    public function update($cid)
+        $id = rand(1, 20);
+        $userId = rand(1, 20);
```

Comment:
You can not use a random id to identify a user that is meant to be in a system.

Reply:
Am not the one that is using the update function or wrote it